src/main/java/com/aws/greengrass/mqtt/bridge/MQTTBridge.java

```diff
@@ -133,14 +131,30 @@ public void startup() {
         }
     }
 
+    // TODO is there a better way than disconnecting?
+    private void restartLocalMqttClient() throws MessageClientException {
```

Comment:
I don't think we want to do this here. The previous implementation had this within the mqtt client, which lets it be smarter about how to handle the change. For example, we don't want to unsubscribe from all topics like we may want to when shutting down. Reconnect should be a fairly fast process that doesn't involve any other changes.